js/modules/handleBurgerButtons.js

```diff
@@ -0,0 +1,18 @@
+const menuOverlay = document.body.querySelector(".menu_overlay");
+const burgerBtn = document.body.querySelector(".menu_burger");
+const menuCloseBtn = menuOverlay.querySelector(".menu_close_btn");
+const menuBody = menuOverlay.querySelector(".menu_body");
+
+burgerBtn.addEventListener("click", function(){
+
+	menuOverlay.classList.add("active");
+	menuBody.classList.add("active");
```

Comment:
you don't need an `active` class for meny body, just use parent selector in css

`.menu_overlay.active .menu_body{ ... }

Reply:
Done.